### DISCUSSION TITLE
Loading the operating system into the repository

### DISCUSSION THREAD

**Discussion by LuLikBP** [community/community#47179] [New to GitHub] [answered]
Hello everyone, I have a question. 

I made my Ubuntu 20.04 compilation with ROS 2 preinstalled and other robotics programs. Is there any way I can upload an iso file or unpack it in the repository?

**Comment by jge162** [comment]
You can upload it to a repo and share the repo link with whomever you deem necessary 100%.

**Reply by genfox27** [reply]
operation:access

**Reply by hannahabudde24** [reply]
> You can upload it to a repo and share the repo link with whomever you deem necessary 100%.

**Comment by Juanescacha** [comment, marked as answer]
well you could upload as @jge162 says you, the .iso to the repo packed or unpacked and share it, but github is more for sharing source code, so if you have a heavy file to share use another cloud service like dropbox, drive, mediafire, etc

**Reply by queenofcorgis** [reply]
Thanks @Juanescacha and @jge162 for offering up some great answers. Appreciate you marking the answer @LuLikBP 🙇🏼

**Comment by Saudimirshid** [comment]
Thank

**Comment by hannahabudde24** [comment]
This is great I need to do this my phone is crazy
